src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -376,6 +376,11 @@ def _validate_vm_create_storage_profile(namespace, for_scale_set=False):
         namespace.attach_os_disk = _get_resource_id(
             namespace.attach_os_disk, namespace.resource_group_name, 'disks', 'Microsoft.Compute')
 
+    if getattr(namespace, 'attach_data_disks', None):
```

Comment:
Is there a reason you are using getattr instead of just `if namespace.attach_data_disks`?